extension.js

```diff
@@ -169,12 +171,21 @@ function assertNextJSApp(componentPath) {
  */
 async function executeCommand(commandInput, componentPath) {
 	const [command, ...args] = shellQuote.parse(commandInput);
-	const cp = child_process.spawn(command, args, {
+	const env = { ...process.env, PATH: `${process.env.PATH}:${componentPath}/node_modules/.bin` };
+	let cp = child_process.spawn('which', [command], { cwd: componentPath, env, stdio: 'ignore' });
+	let [exitCode] = await events.once(cp, 'exit');
+	if (exitCode !== 0) {
+		throw new Error(`Command \`${command}\` not found. Ensure it is included within process.env.PATH`);
+	}
+	cp = child_process.spawn(command, args, {
 		cwd: componentPath,
+		env,
 		stdio: logger.log_level === 'debug' ? 'inherit' : 'ignore',
 	});
-
-	const [exitCode] = await events.once(cp, 'exit');
+	cp.on('error', (error) => {
+		throw error;
```

Comment:
This is going to be an uncaught error, might be preferable to wrap this block in a Promise constructor so we could `reject` here.